{
	"id": "AVG-649",
	"advisories": [
		{
			"content": {
				"id": "ASA-201803-5",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/ASA-201803-5"
					}
				]
			},
			"ecosystems": [
				"arch"
			]
		},
		{
			"content": {
				"id": "AVG-649",
				"severity": [
					{
						"type": "vendor",
						"source": "security.archlinux.org",
						"vendor": "Medium"
					}
				],
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/AVG-649"
					}
				]
			},
			"ecosystems": [
				"arch"
			]
		}
	],
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2018-7536",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-7536"
					}
				]
			},
			"ecosystems": [
				"arch"
			]
		},
		{
			"content": {
				"id": "CVE-2018-7537",
				"references": [
					{
						"source": "security.archlinux.org",
						"url": "https://security.archlinux.org/CVE-2018-7537"
					}
				]
			},
			"ecosystems": [
				"arch"
			]
		}
	],
	"detection": [
		{
			"ecosystem": "arch",
			"criteria": {
				"operator": "OR",
				"criterions": [
					{
						"vulnerable": true,
						"package": {
							"name": "python-django"
						},
						"affected": {
							"type": "pacman",
							"range": [
								{
									"lt": "1.11.11-1"
								}
							],
							"fixed": [
								"1.11.11-1"
							]
						}
					}
				]
			}
		}
	],
	"data_source": "arch"
}
